src/components/order-service/order-service.html

```diff
@@ -75,7 +75,7 @@
         type="button"
         class="btn btn-primary wizard-pf-next"
         ng-if="$ctrl.currentStep.id === 'configure'"
-        ng-disabled="!$ctrl.selectedProject || $ctrl.currentStep.id === 'results'"
+        ng-disabled="!$ctrl.selectedProject || $ctrl.currentStep.id === 'results' || $ctrl.invalidNewProject()"
```

Comment:
Better to have a nested `ng-form` for new project and make that form invalid. Then we can check form validity similar to 

https://github.com/openshift/origin-web-catalog/pull/78/files#diff-a816c7d825f3b359053f0814dcc271f2R32